intel_models/index.md

```diff
@@ -139,7 +139,8 @@ Deep Learning models for text detection in various applications
 
 | Model Name                                                                                                                                     | Complexity (GFLOPs)  | Size (Mp)  |
 |------------------------------------------------------------------------------------------------------------------------------------------------|--------------------- |----------- |
-| [text-detection-0002](./text-detection-0002/description/text-detection-0002.md)                                                                | 51.256               | 6.747      |
+| [text-detection-0003](./text-detection-0003/description/text-detection-0003.md)                                                                | 51.256               | 6.747      |
+| [text-detection-0004](./text-detection-0003/description/text-detection-0004.md)                                                                | TBD                  | TBD        |
```

Comment:
@Ilya-Krylov there is a bug in the path, CI job failed due to that
19:20:26  FILE: index.md
19:20:27  [✖] ./text-detection-0003/description/text-detection-0004.md
19:20:27  
19:20:27  ERROR: dead links found!
19:20:27  [Pipeline] echo
19:20:27  [Error] Found broken links in index.md